tasks/spark-hive-site.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Deploy hive-site.xml in spark's conf directory
+  template:
+    src: "templates/hive-site.xml.j2"
+    dest: "{{spark_usr_dir_orig}}/conf/hive-site.xml"
```

Comment:
Tests are failing because `spark_usr_dir_orig` isn't defined. Should this be `spark_usr_dir`?

Reply:
Ohh shizzle, I thought I had already contributed following var:
```
  spark_usr_dir_orig: "{{spark_usr_parent_dir}}/spark-{{spark_version}}"
```
Motivation: To support deploying (to) multiple spark versions, you should never deploy to`spark_usr_dir`  (but use that only to deploy the final symlink)

Of course, a simple fix here is again expanding to (and introduce the new var later, and use where appropriate!):
```
    dest: "{{spark_usr_parent_dir}}/spark-{{spark_version}}/conf/hive-site.xml"
```
ps: the same change I had contributed already time ago, in deploying spark-env.sh,spark-defaults

Reply:
pushed the fix